src/jupyter_contrib_nbextensions/nbextensions/dragdrop/main.js

```diff
@@ -73,15 +73,14 @@ define([
             data : JSON.stringify(data),
             headers : {'Content-Type': 'text/plain'},
             async : false,
-            success : function (data, status, xhr) {
+            error : function() {console.log('Data transfer for drag-and-drop failed.'); }
```

Comment:
This will get ignored - see [notebook/static/base/js/utils.js#L718](https://github.com/jupyter/notebook/blob/4727efcaf8f8993715831358c1e6b23dad607753/notebook/static/base/js/utils.js#L718)